languages/cs.ini

```diff
@@ -573,13 +600,16 @@ hold_place = "Zadat požadavek"
 hold_place_fail_missing = "Váš požadavek selhal. Chybí některé údaje. Kontaktujte pracovníky knihovny pro další informace."
 hold_place_success_html = "Váš požadavek byl úspěšně vložen. <a href="%%url%%">Vaše rezervace a objednávky</a>."
 hold_profile_html = "Pro informace o rezervacích a objednávkách musíte mít vytvořený <a href="%%url%%">účet v katalogu</a>."
+hold_proxied_by = "Rezervaci vytvořil"
+hold_proxied_for = "Rezervace je určena pro"
 hold_queue_position = "Vaše pořadí ve frontě"
 hold_record_already_on_loan = "Tento dokument již máte rezervován."
```

Comment:
"Na tento dokument již máte rezervaci."

Reply:
I am keeping it as it is - the meaning is the same. But I think it is better to use "action" verb instead of  form with auxiliary verb.